pkg/generate/crd/crd.go

```diff
@@ -26,58 +26,50 @@ import (
 	"github.com/ghodss/yaml"
 	"github.com/spf13/afero"
 	extensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
-	"k8s.io/gengo/args"
+
 	"k8s.io/gengo/types"
-	crdutil "sigs.k8s.io/controller-tools/pkg/crd/util"
 	"sigs.k8s.io/controller-tools/pkg/internal/codegen"
 	"sigs.k8s.io/controller-tools/pkg/internal/codegen/parse"
-	"sigs.k8s.io/controller-tools/pkg/util"
+	util "sigs.k8s.io/controller-tools/pkg/util"
 )
 
-// Generator generates CRD manifests from API resource definitions defined in Go source files.
-type Generator struct {
+// Options receive optional arguments from command line
+type Options struct {
 	RootPath          string
 	OutputDir         string
 	Domain            string
 	Namespace         string
 	SkipMapValidation bool
-
+	apisPkg           string
```

Comment:
The previous comment was very useful for code understanding. Is there a reason we are deleting that ?

Reply:
OK. I will keep the comments.